toktok/about.md

```diff
@@ -0,0 +1,54 @@
+---
+title: About
+permalink: about.html
+menu_index: 6
+navicon: fas fa-align-left
+---
+
+# The Tox Project
+
+Tox began a few years ago, in the wake of Edward Snowden's leaks regarding NSA
+spying activity. The idea was to create an instant messaging application that
+ran without requiring the use of central servers. The system would be distributed,
+peer-to-peer, and end-to-end encrypted, with no way to disable any of the encryption
+features; at the same time, the application would be easily usable by the
+layperson with no practical knowledge of cryptography or distributed systems.
+During the Summer of 2013 a small group of developers from all around the globe
+formed and began working on a library implementing the Tox protocol.
+The library provides all of the messaging and encryption facilities, and
+is completely decoupled from any user-interface; for an end-user to make
+use of Tox, they need a Tox client. Fast-forward a few years to today, and
+there exist several independent Tox client projects, and the original
+Tox core library implementation continues to improve. Tox (both core
+library and clients) has thousands of users, hundreds of contributors,
+and the project shows no sign of slowing down.
+
+Tox is a FOSS (Free and Open Source) project. All Tox code is open source
+and all development occurs in the open. Tox is developed by volunteer
+developers who spend their free time on it, believing in the idea of the
+project. Tox is not a company or any other legal organization. Currently
+we don't accept donations as a project, but you are welcome to reach out
+to developers individually.
+
+# Contact
+
+This website and other infrastructure such as wiki, blog, etc. are maintained
+by some of the long-time contributors of Tox. If you have a legal question,
+or if you notice something is wrong with tox.chat servers, or you find an
+outdated information on the one of tox.chat websites -- you can send an
+email to our [distribution list](mailto:leadership@tox.chat). Please note that this
+list is not the right place to ask for general user support. All such
+emails would simply be ignored. Please use
+our [mailing lists](https://lists.tox.chat/listinfo)
+or the [Reddit forums](https://www.reddit.com/r/projecttox/) for general user support.
+
+For the reference, here is the list of the said long-time Tox contributors
+running the infrastructure:
+
+*   [Impyy](mailto:impyy@tox.chat)
```

Comment:
What is this page about? It's not about the TokTok project, it seems.

Reply:
its a close copy of the about page of tox.chat site.
it's supposed to be a start point, to move the site even closer together.
we can start from here, and then slowly correct anything thats no longer correct